manifests/deploy/openshift/helm-operator/metering.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: metering.openshift.io/v1alpha1
 kind: Metering
 metadata:
   name: operator-metering
-spec:
+spec: []
```

Comment:
This file is generated, and the value used `[]` is incorrect, as the spec is an object, `{}`.

You'll want to set `cr.spec` to `{}` in https://github.com/operator-framework/operator-metering/blob/master/charts/helm-operator/values.yaml#L3-L4

Eg:

```
cr:
  create: false
  spec: {}
```
Then run `make metering-manifests` and commit your changes.

Reply:
@chancez I did that but the generated file is still empty